<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAccountDialog</class>
 <widget class="QDialog" name="FindAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재블럭체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회원계정 찾기</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="box_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY나무L</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>아이디 찾기</string>
   </property>
   <widget class="QPushButton" name="IdBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>양재백두체B</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>아이디 찾기</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="NameLabel">
       <property name="font">
        <font>
         <family>MD개성체</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>이름</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="NameEdit">
       <property name="font">
        <font>
         <family>HY나무L</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY나무L</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>비밀번호 찾기</string>
   </property>
   <widget class="QPushButton" name="PwBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>양재백두체B</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>비밀번호 찾기</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>281</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="IdLabel">
       <property name="font">
        <font>
         <family>MD개성체</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>아이디</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="PhoneLabel">
       <property name="font">
        <font>
         <family>MD개성체</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>핸드폰번호</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="HpEdit">
       <property name="font">
        <font>
         <family>HY나무L</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="IdEdit">
       <property name="font">
        <font>
         <family>HY나무L</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY산B</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>뒤로</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
